### PR TITLE
[ADD] accounting: trusted accounts (send money)

### PR DESCRIPTION
**Pull request by jcs-odoo**
RD task-3210415
Doc task-3211601

### CODE REVIEW

File: content/applications/finance/accounting/payables/pay/trusted_accounts.rst [hunk 1]
```diff
@@ -0,0 +1,49 @@
+=============================
+Trusted accounts (send money)
+=============================
+
+Phishing attacks
+================
+
+A **phishing attack** is an online scam designed to trick individuals or companies into giving away
+sensitive information or money by sending fraudulent communication. Fraudsters pretend to be
```
Comment (toaa-odoo) [inline content/applications/finance/accounting/payables/pay/trusted_accounts.rst:9]:
```suggestion
sensitive information or money by sending fraudulent communication. Fraudsters pretend to be
```
Can you send communication ?

File: content/applications/finance/accounting/payables/pay/trusted_accounts.rst [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+Communication style
+  Fraudulent emails and invoices often use a different communication style, such as **different
+  wording**, and may include **spelling and grammatical mistakes**. Examine and **compare** them
+  with previous ones that you know to be authentic (e.g., payment instructions, language, company
+  logo, etc.).*
+
+Urgency
+  Invoice frauds often use **urgent or threatening language** and change the **payment deadline**.
+  Remain calm and verify if you really received a late payment reminder previously.
```
Comment (toaa-odoo) [inline content/applications/finance/accounting/payables/pay/trusted_accounts.rst:38]:
```suggestion
  Remain calm and check if you really received a late payment reminder previously.
```
Verify might come off as strange to natives here

**Review by toaa-odoo — APPROVED** [review]
Just two weird formulations, all good otherwise

**Comment by jcs-odoo** [comment]
@robodoo r+

**Pull request closed without merging**